tests/testthat/_snaps/age_model_class.md

```diff
@@ -0,0 +1,16 @@
+# can run simulation for the age_model
+
+    [1] "list"
+    [1] "list"
+      time  S1  S2 E1 E2 I1 I2 R1 R2 Incidence.1 Incidence.2
```

Comment:
How about we return the output in long form. So we'd have one column that is time, one that is compartment (S, E, I, R, D, and some letter for incidence), one for age group which gives the actual age range being considered and another for value? That would save us having to use the more arbitrary S1, S2 etc...

Reply:
It also makes it easier for plotting.